.github/workflows/spelling.yml

```diff
@@ -0,0 +1,22 @@
+name: django CMS spelling
+
+on: [push, pull_request]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  codespell:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+          cache: 'pip'
+      - run: python -m pip install codespell
```

Comment:
The codespell action runs in a Docker container which makes these lines unnecessary.
```suggestion
```

Reply:
Thanks @cclauss 👍